jupyterhub/handlers/base.py

```diff
@@ -51,6 +51,20 @@
 class BaseHandler(RequestHandler):
     """Base Handler class with access to common methods and properties."""
 
+    async def prepare(self):
+        """Identify the user during the prepare stage of each request
+
+        During requests, the current user may be retrieved via
```

Comment:
During prepare stage of request, the user is identified by get_current_user. 
During a request, the current...